pkg/volume/api/api.go

```diff
@@ -0,0 +1,388 @@
+package api
```

Comment:
what do you see on git status when you did git mv?

Reply:
I think it is expected that we're making copies here. We have to keep the old files around. Git doesn't have a separate notion of how the files between commits came to be beyond just intelligently diffing commits and inferring what happened